.github/workflows/ci-web.yml

```
@@ -144,3 +144,15 @@ jobs:
           TESTBED_SERVER_URL: parsec3://localhost:6777?no_ssl=true
         working-directory: client
         timeout-minutes: 20
+
+      - name: Install Playwright dependencies
+        run: npx playwright install
```

```suggestion
        run: npx playwright install --with-deps
```

https://playwright.dev/docs/ci-intro#on-pushpull_request